### PR TITLE
Add keytool functiality

### PR DESCRIPTION
**Pull request by fhe3-dh**
Add ability to manage (add only) own CA certificates in installed java

### CODE REVIEW

File: sun-java/settings.sls [hunk 1]
```diff
@@ -26,7 +27,7 @@
   {%- set default_source_hash = 'sha256=1845567095bfbfebd42ed0d09397939796d05456290fb20a83c476ba09f991d3' %}
   {%- set group = 'root' %}
   {%- set archive_type = g.get('archive_type', p.get('archive_type', 'tar' )) %}
-  {%- set java_home = salt['grains.get']('java_home', salt['pillar.get']('java_home', '/usr/lib/java')) %}
+  {%- set java_home = salt['grains.get']('java_home', salt['pillar.get']('java_home', '/usr/lib/jvm')) %}
```
Comment (noelmcloughlin) [inline sun-java/settings.sls]: Leave this alone or propose the change in a separate PR with explanation. I'm not sure of impact on installed base.

**Comment by fhe3-dh** [comment]
Works for me. Please apply with proposed change. Because java_home can be chsnged with pillard, we don't need to change default

**Comment by fhe3-dh** [comment]
I reverted this change in our fork. It is back now to upstream default '/usr/lib/java'

**Comment by aboe76** [comment]
@fhe3-dh and @noelmcloughlin merged

**Pull request merged**